ruby_on_rails/rails_basics/routing.md

```diff
@@ -187,7 +187,7 @@ You should have a good sense of what's going on in the routes file by now but pr
 
 <div class="lesson-content__panel" markdown="1">
 1. Read the [Rails Guides chapter on Routing](http://guides.rubyonrails.org/routing.html), sections 1-2.5, 3.1-3.4, 4.6, and 6.1
-2. Watch this [Wonderful explanation of how REST and HTTP works](https://www.youtube.com/watch?v=Q-BpqyOT3a8). You can follow the tutorial using `curl https://api.github.com`.
+2. Watch this [Wonderful explanation of how REST and HTTP works](https://www.youtube.com/watch?v=Q-BpqyOT3a8). You can follow the tutorial using `curl https://api.github.com` (Note: Postman Chrome Extension is deprecated, use the native [Postman App](https://www.postman.com/downloads/) or use the Web version).
```

Comment:
We can use  our [note box]() styling here.

```suggestion
2. Watch this [Wonderful explanation of how REST and HTTP works](https://www.youtube.com/watch?v=Q-BpqyOT3a8). You can follow the tutorial using `curl https://api.github.com`.

<div class="lesson-note lesson-note--tip" markdown="1">

The Postman Chrome Extension shown in the video is deprecated. Use the native [Postman App](https://www.postman.com/downloads/) or use the web version.

</div>
```
This will render the content like:
![Selection_420](https://github.com/TheOdinProject/curriculum/assets/88392688/a25b7aad-5d67-4474-91fb-21d2cd94ae48)